lib/ansible/modules/cloud/cloudstack/cs_instance_facts.py

```diff
@@ -203,6 +208,7 @@ def __init__(self, module):
             'isoname':              'iso',
             'templatename':         'template',
             'keypair':              'ssh_key',
+            'volumes':              []
```

Comment:
Please remove this line, it is used for replacing the keys for the return values of the virtualmachine API response. this is not meant to set a default.